aviary/mission/flops_based/ode/test/test_landing_ode.py

```diff
@@ -13,11 +14,13 @@
 class FlareODETest(unittest.TestCase):
     def test_case(self):
         prob = om.Problem()
-
         time, _ = detailed_landing_flare.get_item('time')
         nn = len(time)
         aviary_options = inputs
 
+        default_mission_subsystems = get_default_mission_subsystems(
```

Comment:
We kind of don't really even need these lines here, by the look of it. the old default_mission_subsystems as an unused import.